log2sk command is broken

```
$ ./bin/log2sk ../signalk-autostate/test/logs/skserver-raw_2019-09-21T12.log 
{ Error: Cannot find module '@signalk/streams/Simple'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:548:15)
    at Function.Module._load (internal/modules/cjs/loader.js:475:25)
    at Module.require (internal/modules/cjs/loader.js:598:17)
    at require (internal/modules/cjs/helpers.js:11:18)
    at createPipeElement (/home/ihmis-suski/projects/meri-imperiumi/signalk-server-node/lib/pipedproviders.js:56:25)
    at result.pipeElements.providerConfig.pipeElements.reduce (/home/ihmis-suski/projects/meri-imperiumi/signalk-server-node/lib/pipedproviders.js:26:30)
    at Array.reduce (<anonymous>)
    at Object.createPipedProvider (/home/ihmis-suski/projects/meri-imperiumi/signalk-server-node/lib/pipedproviders.js:24:55)
    at Object.<anonymous> (/home/ihmis-suski/projects/meri-imperiumi/signalk-server-node/bin/log2sk:16:43)
    at Module._compile (internal/modules/cjs/loader.js:654:30) code: 'MODULE_NOT_FOUND' }
{ Error: Cannot find module '/home/ihmis-suski/projects/meri-imperiumi/signalk-server-node/lib/../providers/Simple'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:548:15)
    at Function.Module._load (internal/modules/cjs/loader.js:475:25)
    at Module.require (internal/modules/cjs/loader.js:598:17)
    at require (internal/modules/cjs/helpers.js:11:18)
    at createPipeElement (/home/ihmis-suski/projects/meri-imperiumi/signalk-server-node/lib/pipedproviders.js:61:29)
    at result.pipeElements.providerConfig.pipeElements.reduce (/home/ihmis-suski/projects/meri-imperiumi/signalk-server-node/lib/pipedproviders.js:26:30)
    at Array.reduce (<anonymous>)
    at Object.createPipedProvider (/home/ihmis-suski/projects/meri-imperiumi/signalk-server-node/lib/pipedproviders.js:24:55)
    at Object.<anonymous> (/home/ihmis-suski/projects/meri-imperiumi/signalk-server-node/bin/log2sk:16:43)
    at Module._compile (internal/modules/cjs/loader.js:654:30) code: 'MODULE_NOT_FOUND' }
/home/ihmis-suski/projects/meri-imperiumi/signalk-server-node/lib/pipedproviders.js:34
        result.pipeElements[result.pipeElements.length - 1].pipe(new DevNull());
                                                            ^

TypeError: Cannot read property 'pipe' of undefined
    at Object.createPipedProvider (/home/ihmis-suski/projects/meri-imperiumi/signalk-server-node/lib/pipedproviders.js:34:61)
    at Object.<anonymous> (/home/ihmis-suski/projects/meri-imperiumi/signalk-server-node/bin/log2sk:16:43)
    at Module._compile (internal/modules/cjs/loader.js:654:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:665:10)
    at Module.load (internal/modules/cjs/loader.js:566:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:506:12)
    at Function.Module._load (internal/modules/cjs/loader.js:498:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:695:10)
    at startup (internal/bootstrap/node.js:201:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:516:3)
```

--- a/bin/log2sk
+++ b/bin/log2sk
@@ -16,7 +16,7 @@ const app = {
 new require('../lib/pipedproviders')(app).createPipedProvider({
   pipeElements: [
     {
-      type: 'providers/Simple',
+      type: 'providers/simple',
       options: {
         logging: false,
         noThrottle: true,

